specification/reservations/resource-manager/Microsoft.Capacity/preview/2019-04-01/reservations.json

```diff
@@ -1012,6 +1012,9 @@
         },
         "renewProperties": {
           "$ref": "#/definitions/RenewPropertiesResponse"
+        },
+        "term": {
+          "$ref": "#/definitions/ReservationTerm"
```

Comment:
>ReservationTerm [](start = 33, length = 15)

You are adding a new property non-readonly property to a stable API version. This is considered a breaking change. You should introduce this change in a new API version.